reporters_db/data/laws.json

```diff
@@ -5191,14 +5310,48 @@
             ],
             "jurisdiction": "United States",
             "name": "United States Statutes at Large",
-            "notes": "Automatically generated. Remove this message if replacing with real examples.",
+            "notes": "Some of these citations may be to treaties, which were reported here until December 31, 1949. See Ch 1001, Sec. 2, 64 Stat. 980 (1950).",
             "regexes": [
                 "$volume $reporter $page_with_commas"
             ],
             "start": null,
             "variations": []
         }
     ],
+    "T.I.A.S.": [
+        {
+            "cite_type": "treaty",
+            "end": null,
+            "examples": [
+                "T.I.A.S. No. 6577"
+            ],
+            "href": "https://law.resource.org/pub/us/code/blue/IndigoBook.html#T6",
+            "jurisdiction": "United States",
+            "name": "Treaties and Other International Acts Series",
+            "regexes": [
+                "$reporter\\s*(?:No\\.|Number)?\\s*(?P<number>\\d+)"
+            ],
+            "start": "1945-01-01T00:00:00",
+            "variations": []
+        }
+    ],
+    "T.S.": [
+        {
+            "cite_type": "treaty",
+            "end": "1945-12-31T23:59:59",
+            "examples": [
+                " T.S. No. 251"
```

Comment:
extra whitespace.

Reply:
That was on purpose. Some other citations end in T.S. Without the required space, C.E.T.S., for example, would match. There may be a more elegant way to do this but requiring the whitespace seemed simplist to me.

Reply:
Wouldn't the general tokenizer in eyecite handle the token so that `C.E.T.S.` wouldn't get matched?

Reply:
Good point. I had not thought of that. I will remove the white space.